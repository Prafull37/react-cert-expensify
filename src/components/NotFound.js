import React from 'react';
import {Link} from 'react-router-dom';

const NotFound=(props)=>(<div>
    <div>404 not match- <Link to="/">Go Home</Link></div>
</div>)


export default NotFound;