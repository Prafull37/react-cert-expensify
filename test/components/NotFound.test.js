import React from "react"
import {shallow} from "enzyme";
import NotFound from "../../src/components/NotFound";

test('should render NotFound correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });