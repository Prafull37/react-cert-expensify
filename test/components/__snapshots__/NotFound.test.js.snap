// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<div>
  <div>
    404 not match- 
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </div>
</div>
`;
